content/en/tracing/setup_overview/custom_instrumentation/cpp.md

```diff
@@ -69,15 +69,20 @@ To manually instrument your code, install the tracer as in the [setup examples][
 
 ```cpp
 {
-  // Create a root span.
-  auto root_span = tracer->StartSpan("operation_name");
-  // Create a child span.
+  // Create a root span for the request we're currently processing.
```

Comment:
```suggestion
  // Create a root span for the processing request.
```

Reply:
Perhaps this is better:

> // Create a root span for the current request.

The verb "processing" is a little awkward and ambiguous here.

Reply:
Ah, my browser was showing me an older version, before this change was merged.